src/main/java/com/hivemq/cli/commands/shell/ShellCommand.java

```diff
@@ -127,12 +128,8 @@ public void run() {
 
 
     private void interact() {
-        shellCommandLine = new CommandLine(spec);
-        contextCommandLine = DaggerContextCommandLine.create().contextCommandLine();
-
-        shellCommandLine.setColorScheme(MqttCLIMain.COLOR_SCHEME);
-        contextCommandLine.setColorScheme(MqttCLIMain.COLOR_SCHEME);
-        contextCommandLine.setUsageHelpWidth(MqttCLIMain.CLI_WIDTH);
+        shellCommandLine = DaggerMqttCLI.create().commandLine().getSubcommands().get("shell");
```

Comment:
I don't think it is intended to create a new Dagger component and a new command line here, is it?

Reply:
maybe use a static field in MqttCLI which stores `DaggerMqttCLI.create()`